server-stuff.md

```diff
@@ -0,0 +1,61 @@
+
+## Using a Personal Backend
+
+This is concidered an advanced function and is unnecessasary for normal usage
+
+The default for ovos-core is no backend.
+
+You can go without a backend and go offline and use our free proxy for API services with no accounts.
```

Comment:
Consider a sentence or two about what the privacy and data implications are of using OVOS proxy services.